overlord/configstate/configcore/services.go

```diff
@@ -62,7 +62,7 @@ func switchDisableService(service, value string) error {
 }
 
 // services that can be disabled
-var services = []string{"ssh", "rsyslog"}
+var services = []string{"ssh", "rsyslog", "snapd.autoimport"}
```

Comment:
Yeah, this is definitely not appropriate, per note from @pedronis. It's an implementation detail that there is a systemd service named like this at all, and I'm not even sure that today this is indeed the only way we run assertion auto-imports.

Before going into alternatives, who's asking for this and why?

Reply:
a customer with a drone project is asking for this... their HW has a few disk devices attached and the job probes them, they want to reduce any non-required activity on boot (fast boot/reboot time is critical for them) and will not make use of this feature which is why they asked for a way to turn it off. To prevent anyone from using hackish customization scripts to achieve this, a core config option seems like the sanest solution and using the service config option had simply the least impact here ... If you want to use a new namespace instead thats surely possible ... though simply not documenting it for public use ... like we do with the core-support interface would probably just do as well.

Reply:
Oliver, you say "not document for public use" in the same sentence you say "I want to use this in a customer". You see how conflicting this is?

Please send us more details, in the forum or elsewhere, about what is the actual impact that this service is causing. If we have no good reason to disable the service, we shouldn't disable it. This is not a timer.. it won't do anything just because it is enabled.

Reply:
I'm not sure what is wrong about "not documented but still usable for a commercial product via a gadget setting" 

I opened a forum topic about this two weeks ago in communication with michael (linked in the PR description above) and got no further reaction to it, which is what i counted as consent. 

The feature probes all attached devices, mounts them and checks for possible assertions, disabling it causes a lot less load on boot, the difference has been tested and there are scripts in the product today (added by the customer) that manually disable the job to overcome the issue. The plan was to transition this into something non-hackish with your teams help to improve the product. As i said on IRC, I'll happily work on any alternative, be it a proxying function to move it to another config namespace or anything else that you suggest to make this a sane feature.

Reply:
there are's another part involved with auto-import of assertions,

data/udev/rules.d/66-snapd-autoimport.rules

are they disabling that too?

I looked again at snap auto-import without arguments, afaict it reads /proc/self/mountinfo, and does file exists on 1 file in each device, still a bit unsure why it would provoke alone a lot of load cc @mvo5

Reply:
The udev rule will not have any impact on boot, all "add" uevents for pre-plugged disk devices in the kernel get processed by the first udevd started in the initrd where we do not ship the rule. Once the second udevd gets started (after switching to the rootfs) they have already been processed. The rule will only have impact on additionally plugged in devices after boot (unless we move it over to teh initrd along with the executables it calls from the RUN parameter) .

That said ... to disable the feature completely, the rule should indeed also not be in use.